server/api/paths.js

```diff
@@ -2,6 +2,7 @@ let neo4j = require('neo4j-driver').v1;
 let driver = neo4j.driver("bolt://localhost", neo4j.auth.basic("neo4j", "1234"))
 let session = driver.session();
 const router = require('express').Router()
+const dummyPaths = require('../../script/data/path.js')
```

Comment:
More dummy content. Not sure if this is needed anymore.

Reply:
same